### PR TITLE
P1/feat/24 categorized carousel

### PR DESCRIPTION
**Pull request by netlightebba**
Filter functionality for carousel.

### CODE REVIEW

**Review by netlight-johannadagfalk — COMMENTED** [review]
The rendering of the site works weird when pressing some of the filters (not all). There is sometimes a sudden jump to another place on the site that is confusing for the user. What could be the reason for this?

**Comment by netlightebba** [comment]
> The rendering of the site works weird when pressing some of the filters (not all). There is sometimes a sudden jump to another place on the site that is confusing for the user. What could be the reason for this?

Yes - I know :( It is a problem with the rerender of the carousel I believe. I would like to keep the "bad" behavior until the carousel is exchanged to  adapt the solution to the new carousel.

File: app/assets/main/react_components/User/ClimateActionsContainer.jsx [hunk 1]
```diff
@@ -1,142 +1,165 @@
...
+      ? totClimateActions.filter(
+          (temp) => temp.climate_action_category_id === cat
+        )
+      : totClimateActions;
 
-  const monthlyAction = climateActionsUser.find(
-    (action) => action.action_of_the_month === true,
-  )
+    const filteredActionsWithStatus = filteredActions.map((x) => {
+      return totUserActions.some((y) => y.climate_action_id === x.id)
```
Comment (elinaolsson) [inline app/assets/main/react_components/User/ClimateActionsContainer.jsx:133]: A bit hard to interpret x and y, maybe change name to something including the word "action"

**Review by elinaolsson — APPROVED** [review]
Great job! 🤙

**Review by netlight-johannadagfalk — APPROVED** [review]
👯

**Pull request merged**